frontend/packages/integration-tests-cypress/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - rhamilto
+  - TheRealJon
+approvers:
+  - spadgett
```

Comment:
You can take me off here since I'm already an approver in the root OWNERS file.

Reply:
fixed